### ISSUE TITLE
PlayControlButton: types for `globalKeyboardListener` and `event`? 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#731]
I’m converting scenery-phet PlayControlButton to TS.  I can’t figure out the types for `globalKeyboardListener` and `event` (see red underlines) in this chunk of code:  

<img width="906" alt="screenshot_1644" src="https://user-images.githubusercontent.com/3046552/161894040-b04ccefd-304d-4a01-9457-1d2e79becfc0.png">

I'm pretty sure (but not certain) that `event` should be `( event: Event )` (native DOM Event).

I've tried a few different things for `globalKeyboardListener`, including:

```js
let globalKeyboardListener: ( ( event: Event ) => void ) | null;
if ( if ( options.includeGlobalHotkey ) {
  globalKeyboardListener = ( event: Event ) => {
    ...
  };
} 
else {
  globalKeyboardListener = null;
}
```

But then TS complains:

> TS2345: Argument of type ‘(event: Event) => void’ is not assignable to parameter of type ‘Listener[]>’

I'm confused because `Listener` is private to Emitter. (Emitter parameterization is very confusing.)

@jessegreenberg @zepumph Any help will be appreciated.

**Comment by pixelzoom** [comment]
I've checked in the partial conversion of PlayControlButton.ts, with `// @ts-ignore` for the `globalKeyboardListener` bits.

**Comment by pixelzoom** [comment]
Ah... Maybe the probem is that KeyStateTracker is still .js, and doesn't have the correct definition for `keyupEmitter`?

**Comment by zepumph** [comment]
This worked for me. Anything else here?

**Comment by pixelzoom** [comment]
Hmmm...  @zepumph it looks like you duplicated my fix before I assigned back to @jessegreenberg for review.  Thought I was the one reviewing this...

**Comment by zepumph** [comment]
Ha! So fun. Ready to close?

**Comment by jessegreenberg** [comment]
Oh thank you, looks good. I had meant to go through and clean up modifiers before closing https://github.com/phetsims/scenery/issues/1400. Changes and https://github.com/phetsims/scenery-phet/issues/731#issuecomment-1091913457 look good to me.
